packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -128,19 +128,22 @@ export class CallContext {
   public removeCall(callId: string): void {
     this.setState(
       produce(this._state, (draft: CallClientState) => {
-        draft.calls.delete(callId);
+        const callIndex = draft.calls.findIndex((candidate) => candidate.id === callId);
+        if (callIndex !== -1) {
+          draft.calls.splice(callIndex, 1);
```

Comment:
because javascript arrays STILL don't have a remove function, would be nice or unecessary to comment instances of this with `// remove element from array` for readers not intimate with javascript array manipulation

Reply:
Will resist this request.
Documenting the behaviour of standard library methods adds clutter.